framework/doc/content/syntax/BCs/Periodic/index.md

```diff
@@ -1,29 +1,47 @@
 # Periodic System
 
-!alert! construction title=Undocumented System
-The Periodic system has not been documented. The content listed below should be used as a starting
-point for documenting the system, which includes the typical automatic documentation associated with
-a system; however, what is contained is ultimately determined by what is necessary to make the
-documentation clear for users.
+Periodic boundary conditions enforce value constraints on separate boundaries in the mesh.
+This capability is useful for modeling quasi-infinite domains and systems with conserved
+quantities.
 
-```markdown
-# Periodic System
+MOOSE has full support for Periodic BCs in
 
-## Overview
+- 1D, 2D, and 3D.
+- With mesh adaptivity.
+- Can be restricted to specific variables.
 
-!! Replace this line with information regarding the Periodic system.
+Supports arbitrary translation vectors for defining periodicity.
```

Comment:
MOOSE supports more than just the translation-by-fixed-vector case, right?  `FunctionPeriodicBoundary` will give you translations by arbitrary parsed functions.

Reply:
ah that s what I meant to convey. i ll reword